ckanext/spatial/harvesters/gemini.py

```diff
@@ -154,6 +154,10 @@ def write_package_from_gemini_string(self, content, harvest_object):
         if last_harvested_object:
             # We've previously harvested this (i.e. it's an update)
 
+            # sets harvest_object.force_import property to self.force_import when set
+            if hasattr(harvest_object, 'force_import'):
+                self.force_import = harvest_object.force_import
```

Comment:
Would it be more Python to catch this an exception, perhaps like:

```suggestion
            try:
                self.force_import = harvest_object.force_import
            except AttributeError:
                pass
```

Reply:
I've updated with `getattr` which should make it clearer

Reply:
I've updated with `getattr` which should make it clearer